src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -39,6 +39,7 @@ internal class AD7DebugSession : DebugAdapterBase, IDebugPortNotify2, IDebugEven
         private IDebugEngine2 m_engine;
         private EngineConfiguration m_engineConfiguration;
         private AD7Port m_port;
+        private string clientId;
```

Comment:
>clientId [](start = 23, length = 8)

Private fields should begin with `m_` or `_` in at least most of our code bases.

Reply:
Noted. Thank you for bringing this up.